L1Trigger/L1TZDC/test/testL1TZDCEtSumsProducer_cfg.py

```diff
@@ -0,0 +1,97 @@
+import FWCore.ParameterSet.Config as cms
+
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(prog=sys.argv[0],
+    description='Test of the L1TZDCEtSumsProducer plugin')
+
+parser.add_argument('-f', '--fileNames', dest='fileNames', nargs='+',
+    default=['/store/hidata/HIRun2024B/HIForward0/RAW/v1/000/388/784/00000/a277c6d8-c445-4d2b-a45c-0e74e4ed8ce8.root'],
+    help='Input EDM file(s)'
+)
+
+parser.add_argument('-l', '--rawDataLabel', dest='rawDataLabel', type=str, default='rawDataRepacker',
+    help="Label of the FEDRawDataCollection product to be used as input")
+
+parser.add_argument('-g', '--globalTag', dest='globalTag', type=str, default='150X_dataRun3_HLT_v1',
```

Comment:
Will change this to `auto:run3_hlt_relval`, so the name of a production GlobalTag is not hard-coded in this config.